### PR TITLE
Added instructions for working with pre-stitched images

### PR DESCRIPTION
**Pull request by ArtemSokolov**
Updated Inputs/Outputs and FAQ sections of the docs, informing users how to apply MCMICRO to pre-stitched and pre-registered images.

### CODE REVIEW

**Comment by ArtemSokolov** [comment]
@jtefftHiTS, @hxu-hms: Applying MCMICRO to pre-stitched and pre-registered images has been an increasingly common use case. I added a couple of small blurbs about working with such images. Will you please review when you have a chance?

**Review by hxu-hms — COMMENTED** [review]
The new addition on using pre-registered images looks great! My only suggestion is for the content in `io.md`. I would rephrase the first part of the Input section right after"**At the minimum, the pipeline expects two inputs**". I recommend changing "2. Raw images..." to "2. ome.tiff images", then specify they have two options for the images. Something to the effect of: 

2. ome.tiff images. You have two options here:
a.  raw images under `raw` directory (link to ##Raw images header below) 
OR
b. pre-registered images  under `registration` directory (link to ##Using pre-registered images header below). #

The current order makes me wonder if I still need the raw images even if I'm adding pre-registered images in `registration`.

**Comment by ArtemSokolov** [comment]
Great idea, @hxu-hms. I used a slight variant of your suggestion, since the raw image tiles don't have to be .ome.tiffs.
